init.g

```diff
@@ -14,13 +14,11 @@ if not IsBound(UserHomeExpand) then
 fi;
 
 # load kernel function if it is installed:
-if (not IsBound(DIGRAPHS_C)) and ("digraphs" in SHOW_STAT()) then
+if "digraphs" in SHOW_STAT() then
```

Comment:
@james-d-mitchell Do you just want the `DIGRAPHS_C` removing? Or replacing with something else/`DIGRAPHS_C` recreating? What is the purpose of it?

Reply:
The purpose is to prevent loading the static module, and then the dynamic one. It could be replaced with a check that any thing defined in the kernel module is bound though, such as `DIGRAPH_OUT_NEIGHBOURS`

Reply:
I'll add something in as you suggest. I get that the second check for `IsBound(whatever)` would stop the dynamic module being loaded immediately after the static module is loaded, but then I don't get why the line check (line 17) is needed. Or is `init.g` re-read sometimes?